test/crud_app_web/controllers/goal_controller_test.exs

```diff
@@ -0,0 +1,51 @@
+defmodule CrudAppWeb.GoalControllerTest do
+  use CrudAppWeb.ConnCase
+
+  alias CrudApp.BucketList
+  alias CrudApp.BucketLis.Goal
```

Comment:
Typo?

Reply:
Removed in https://github.com/hyoyou/crud-app-elixir/pull/18/commits/9dec71ec112d08251fa65c8778cf897732914264